StyleCop.Analyzers/StyleCop.Analyzers/StyleCop.Analyzers.csproj

```diff
@@ -442,6 +443,7 @@
     <EmbeddedResource Include="OrderingRules\OrderingResources.resx">
       <Generator>ResXFileCodeGenerator</Generator>
       <LastGenOutput>OrderingResources.Designer.cs</LastGenOutput>
+      <SubType>Designer</SubType>
```

Comment:
Sometimes Visual Studio adds these. I can remove it when I merge the pull request, if it's still there.